content/tutorials/jwt-authentication.textile

```diff
@@ -33,51 +33,54 @@ meta_keywords: "Ably realtime, JWT, authentication"
 
 This tutorial guides you through the process of creating a simple authentication server that generates a "JSON Web Token (JWT)":/core-features/authentication#token-authentication. You'll see how the client can use this JWT to authenticate with Ably. You will also learn how the client can automatically request a new JWT, before the current JWT expires.
 
-Ably supports two types of authentication schemes: 
+Ably supports two types of authentication schemes:
 
-* "Basic authentication":/core-features/authentication#basic-authentication - this uses one of your private API keys and is designed for use by your servers. 
+* "Basic authentication":/core-features/authentication#basic-authentication - this uses one of your private API keys and is designed for use by your servers.
 * "Token authentication":/core-features/authentication#token-authentication - can be used by your device and browser clients. A short-lived secure token is issued to these clients by your auth server. "JSON Web Token (JWT) authentication":/core-features/authentication#token-authentication is an extension of the token-based authentication scheme in Ably.
 
 When using token authentication, Ably recommends you instantiate the client library with a method to obtain tokens, rather than a token itself. This is due to the fact that tokens eventually expire, resulting in the connection being lost. By providing a method to obtain tokens instead, the Ably client library will automatically get a new token once the current one expires, maintaining a continuous connection.
 
 "JSON Web Tokens":https://jwt.io/ (JWTs) are an open, industry standard (RFC 7519) method for representing claims securely between two parties. JWT auth is especially helpful in scenarios where a single authentication scheme is to be used for all devices and browsers, and third-party platforms such as Ably. JWT auth is also a convenient way to authenticate your IoT devices or servers which are built using the platforms that Ably doesn't yet support. This is possible because you don't have to instantiate an Ably client when using the JWT auth scheme.
 
-Ably offers two different ways in which you can use a JWT to authenticate your users: 
+Ably offers two different ways in which you can use a JWT to authenticate your users:
 
-* You can use the JWT as the token itself by formatting it correctly (called an Ably JWT). 
-* You can pass to Ably a JWT of any structure that contains an Ably Token. 
+* You can use the JWT as the token itself by formatting it correctly (called an Ably JWT).
```

Comment:
Would be good to link to the Ably JWT docs here: https://ably.com/docs/api/realtime-sdk/authentication#ably-jwt

Reply:
nice!